### PR TITLE
Fix bugged cherry fence gate sounds.

### PR DESCRIPTION
**Pull request by letsgoawaydev**
Fixes https://bugs.mojang.com/browse/MCPE-168021

### CODE REVIEW

**Review by onebeastchris — APPROVED** [review]

File: sounds/sound_definitions.json
Comment (onebeastchris) [inline sounds/sound_definitions.json:1]: Can you mention the change here (with a link to the bug report) on the readme/developer information page?

Comment (letsgoawaydev) [reply sounds/sound_definitions.json:1]: Sure, will do soon. Currently fixing https://github.com/GeyserMC/Geyser/pull/4815 for the copper door waxing sound and once i've got that working ill do this

Comment (letsgoawaydev) [reply sounds/sound_definitions.json:1]: Finished the docs now.

File: developer_documentation.md [hunk 1]
```diff
@@ -315,4 +313,72 @@ Hiding the 2x2 crafting grid is a bit more involved. We have to use bindings to
...
+    "sounds" : [
+      "sounds/block/cherry_wood_fence_gate/toggle1",
+      "sounds/block/cherry_wood_fence_gate/toggle2",
+      "sounds/block/cherry_wood_fence_gate/toggle3",
+      "sounds/block/cherry_wood_fence_gate/toggle4"
+    ]
+},      
+```
+
+This `toggle4` sound does not exist in Bedrock Editions's default resources, meaning that sometimes the Cherry Fence Gate can play no sound sometimes.
```
Comment (onebeastchris) [inline developer_documentation.md:357]:
```suggestion
This `toggle4` sound does not exist in Bedrock Editions's default resources, meaning that sometimes the Cherry Fence Gate can play no sound.
```

File: developer_documentation.md [hunk 2]
```diff
@@ -315,4 +313,72 @@ Hiding the 2x2 crafting grid is a bit more involved. We have to use bindings to
...
+
+Both Java and Bedrock dont have this sound, so GeyserOptionalPack fixes this by redefining these sound events, and removing `toggle4` from the `sound_definitions.json` as seen below:
```
Comment (onebeastchris) [inline developer_documentation.md:359]:
```suggestion
Both Java and Bedrock don't have this sound, so GeyserOptionalPack fixes this by redefining these sound events, and removing `toggle4` from the `sound_definitions.json` as seen below:
```

**Review by onebeastchris — APPROVED** [review]

**Comment by letsgoawaydev** [comment]
Typos have been fixed.

**Pull request merged**